front/lib/api/assistant/actions/process.ts

```diff
@@ -41,75 +43,100 @@ import {
 import { AgentProcessAction } from "@app/lib/models/assistant/actions/process";
 import logger from "@app/logger/logger";
 
-/**
- * Model rendering of process actions.
- */
+interface ProcessActionBlob {
+  id: ModelId; // AgentProcessAction.
+  agentMessageId: ModelId;
+  params: {
```

Comment:
`nit` move to a separate type.

Reply:
Since it's a nit I'm merging as is and will discuss it with you tomorrow -> I like having it flat and avoid a click to redirect to another file to understand what's in params. If you're against it I'll handle the refactor for both this one and the one from Process action in a dedicated PR 🙇🏻‍♀️

Reply:
I agree but if we have to write twice the same type it probably means that one type could abstract it, right 🙃?

Reply:
Yes and no but ok to apply it! Will come in a next PR 🙇🏻‍♀️

Reply:
The real real question is why we need 2 types :p